clash-prelude/tests/doctests.hs

```diff
@@ -1,14 +1,7 @@
-{-# LANGUAGE CPP #-}
-
 module Main where
 
-import Build_doctests (flags, pkgs, module_sources)
-import Data.Foldable (traverse_)
-import Test.DocTest (doctest)
+import Test.DocTest (mainFromCabal)
+import System.Environment (getArgs)
```

Comment:
Alphabetical please

Reply:
Are there any linting tools that:

* Check this
* Work during compile time (i.e., generate a warning WITHOUT running something after the fact)

That you are aware of?

Reply:
Not that I'm aware of unfortunately, but I don't think it would be too hard to hack together a check like we have the check for EOL whitespace. It would just have to

  1. `grep` for any line starting with `import`
  2. remove any word `qualified` that appears
  3. check the module names are sorted

So if either of us finds the time we can hack a check together in `bash` and add it to the CI checks. I don't think I'll have time this week, so if you don't can you make an `enhancement` issue for it?

Reply:
Yeah, the reason I asked is that I really "hate" after-the-fact checks. The EOL checks are alright, because it's just an editor setting. Time to test [`styleless-haskell`](https://gitlab.com/martijnbastiaan/styleless-haskell/-/blob/master/src/Plugin/StylelessHaskell/Checks/Alphabetization.hs)? :wink:

Reply:
`haskell-language-server` has `retrie` integration, perhaps we could configure that to do the rewrite in-editor?

Reply:
yes pls